test/components/views/right_panel/RoomHeaderButtons-test.tsx

```diff
@@ -94,6 +95,10 @@ describe("RoomHeaderButtons-test.tsx", function() {
         room.setThreadUnreadNotificationCount("$123", NotificationCountType.Highlight, 0);
 
         expect(container.querySelector(".mx_RightPanel_threadsButton .mx_Indicator")).toBeNull();
+
+        // Thread activity should appear on the icon.
+        mkThread({ room, client, authorId: client.getUserId()!, participantUserIds: ["@alice:example.org"] });
+        expect(isIndicatorOfType(getComponent(room), "bold")).toBe(true);
```

Comment:
This assertion is failing -- from adding some debug logs it seems like the thread is getting created *after* the assertion. I'm not sure what's going on.